packages/components/src/styles/components/application-state.scss

```diff
@@ -7,43 +7,96 @@
 // APPLICATION-STATE COMPONENT
 //
 
-$hds-application-state-padding: 12px 0;
+$hds-application-state-content-max-width: 480px;
+$hds-application-state-footer-child-gap: 8px;
 
 .hds-application-state {
-  width: 19.5rem;
+  display: grid;
+  grid-template-columns: minmax(0, $hds-application-state-content-max-width) 1fr 1fr;
+  grid-template-rows: auto auto auto auto;
```

Comment:
The overall ApplicationState layouts (with left vs. centered alignment) are fairly simple so to me using grid to lay everything out seems overly complex and probably not necessary.

Reply:
This was the best way I could come up with for this alignment without modifying the structure or adding any additional elements.

I did play around with flexbox but didn't come to a conclusion that satisfied the design requirements.

Reply:
Also, the current implementation is not working as expected

Before:
<img width="835" alt="screenshot_3984" src="https://github.com/user-attachments/assets/f0ae0a4a-8d63-4993-baa6-9943612a4e78">

After
<img width="868" alt="screenshot_3983" src="https://github.com/user-attachments/assets/d8c4b9c8-e059-41a7-af9f-d85109f46f51">

This for different reasons:
- a grid layout fills the parent container by default, while the previous implementation had a fixed width
- the grid colums don't match the content

I agree with @KristinLBradley that before jumping to using a CSS Grid, probably an attempt to see if the simple CSS Flexbox would do the trick (I mean for the root container of the ApplicationState)

One suggestion I have is: in the showcase, leave the content as it was before, and add below the "new" content, so you can spot if you're introducing breaking changes in the layout.